mautic/mautic
# add default bcc

| Q  | A
| --- | ---
| Bug fix? |  No
| New feature? |  Yes
| Automated tests included? | No
| Related user documentation PR URL | https://github.com/mautic/documentation/pull/314
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? |  No
| Deprecations? |  No

#### Description:

Add the ability to set a default BCC, so any mail that was not set with one, will get the default one (if set).
1 use case is to save all outgoing emails in a different email account for debugging and history preservation - with this feature it is much easier to achieve without changing 100's of email configurations

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Set a default BCC address in the email configuration section
3. Send out a test email and make sure it was sent to the bcc address as well. ( you will need to configure the email settings to something that will actually send out emails)
4. Create an email template, and set a different bcc email to it
5. Get the email to be sent out (in a flow) and make sure that it was bcc'd to the email specif address and not the default one
